documentation/src/docs/usage/formatLineItems().mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: formatLineItems()
+---
+
+`formatLineItems(cartDetails)` Is a function that helps with creating Session IDs server side
+for products that have been created on the Stripe dashboard. It's preferred to use this with the
+Price API.
+
+
+It's important that your product data has a `price_id` key that provides the given Price ID you get
+from Stripe's dashboard.
```

Comment:
I like the way you put this. The only thing I might add is an example of how to do use it and what an example output might look like. And if I'm remembering right we need to document the `price_data` thing that you coded before.

Reply:
I think I documented it in `addItem` but not in the `API` section, good call

Reply:
I added the `price_data` stuff to Getting Started with a bit of an explanation between Server(less) and client only mode.


I hope this is it!